### PR TITLE
[radar-gateway] Update configmap variables

### PR DESCRIPTION
**Pull request by mpgxvii**
**Description of the change**
- Updates `authorizer.yml` configmap to allow setting of public key urls for token verification (other than MP)
- Updates chart to allow https://github.com/RADAR-base/RADAR-Gateway/pull/110

**Benefits**

**Possible drawbacks**

**Applicable issues**

  - fixes #

**Additional information**

- Depends on gateway release: https://github.com/RADAR-base/RADAR-Gateway/pull/113
**Checklist**

- [ ] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [ ] Variables are documented in the README.md
- [ ] Title of the PR starts with chart name (e.g. [<name_of_the_chart>])

### CODE REVIEW

File: charts/radar-gateway/values.yaml [hunk 1]
```diff
@@ -228,3 +228,7 @@ cc:
   schemaRegistryApiKey: srApiKey
   # -- Confluent Cloud schema registry API secret
   schemaRegistryApiSecret: srApiSecret
+
+public_key_endpoints:
+  # List of public key endpoints for token verification
```
Comment (keyvaann) [inline charts/radar-gateway/values.yaml:233]: The documentation should be above the item

**Review by yatharthranjan — APPROVED** [review]
I have updated the doc and default. LGTM

**Comment by keyvaann** [comment]
Thanks!

**Comment by keyvaann** [comment]
@mpgxvii @yatharthranjan I was trying to apply this update and got this error message:
```
[2024-10-23 16:56:59,026] INFO  - Reading configuration from /etc/radar-gateway/gateway.yml (ConfigLoader.kt:64)
[2024-10-23 16:56:59,526] ERROR - Usage: <command> [/etc/radar-gateway/gateway.yml] (ConfigLoader.kt:82)
[2024-10-23 16:56:59,527] ERROR - Failed to read config file /etc/radar-gateway/gateway.yml: Unrecognized field "publicKeyUrls" (class org.radarbase.gateway.config.AuthConfig), not marked as ignorable (7 known properties: "managementPortalUrl", "keyStore", "resourceName", "authorizeListTopics", "publicKeys", "checkSourceId", "issuer"])
 at [Source: (BufferedInputStream); line: 20, column: 1] (through reference chain: org.radarbase.gateway.config.GatewayConfig["auth"]->org.radarbase.gateway.config.AuthConfig["publicKeyUrls"]) (ConfigLoader.kt:83)
```

**Pull request merged**